src/commands/image/deployImageApi/deployImage.ts

```diff
@@ -45,4 +46,6 @@ export async function deployImage(context: IActionContext & Partial<IContainerRe
 
     await wizard.prompt();
     await wizard.execute();
+
+    void showContainerAppNotification(containerApp, true /** isUpdate */);
```

Comment:
These notifications must respect the 
<img width="386" alt="image" src="https://github.com/microsoft/vscode-azurecontainerapps/assets/12476526/7649089e-c458-4a4a-a2f0-932ecf49ea12">
setting.

Basically if this is on, and we are showing the action in the activity log, we shouldn't also show a notification. You can use this util to do so: https://github.com/microsoft/vscode-azurecontainerapps/blob/d657811a0c652d20c16562dd84ef0ad68dc45fbd/src/utils/activityUtils.ts#L14